vars/mavenCanaryRelease.groovy

```diff
@@ -9,12 +9,16 @@ def call(body) {
 
     def flow = new io.fabric8.Fabric8Commands()
     def utils = new io.fabric8.Utils()
+    def pomFMPVersionKey = config.fmpVersionPropertyName ?: "fabric8-maven-plugin.version"
```

Comment:
does this mean it will use the fmpversionproerty from users pom.xml if present and otherwise default to the specific version ?

Reply:
@maxandersen yes it uses fmpversionproerty from pom.xml. But its always going to use a specific version of FMP between [3.5.38 to 4.0.0](https://github.com/fabric8io/fabric8-pipeline-library/pull/392/files#diff-663252489f1c9712dbeb52401d084235R16) if [autoUpdateFMPVersion](https://github.com/fabric8io/fabric8-pipeline-library/pull/392/files#diff-663252489f1c9712dbeb52401d084235R12) flag is enabled

Reply:
dropping open ended version here, its not working for resolving plugin version.